spec/spec_helper.rb

```diff
@@ -88,7 +88,11 @@ def host_ip
 end
 
 # This makes it visible to others
-Capybara.server_host = host_ip
+if ENV['REMSAND']
```

Comment:
This is required for running tests using remote sandbox. 'REMSAND' variable is set in docker compose files which use remote sandbox.